geommethods.yaml

```diff
@@ -154,6 +154,10 @@
 - name: Length
   comment: returns g's length
   type: float64Property
+- name: MakeValid
+  comment: Repair an invalid geometry, returning a valid output.
```

Comment:
nit: the comment should begin with a lowercase letter and not include punctuation, as these are added by the template.